targets/nvidia-jetson-orin.nix

```diff
@@ -10,14 +10,25 @@
 }: let
   name = "nvidia-jetson-orin";
   system = "aarch64-linux";
+  somDevices = {
+    "agx" = {
+      bus = "pci";
+      path = "0001:01:00.0";
+    };
+    "nx" = {
+      bus = "pci";
+      path = "0008:01:00.0";
+    };
+  };
```

Comment:
did you look at defining these in modules/hardware/nvidia-jetson-orin/jetson-orin.nix, to keep the HW definitions self contained ?

Reply:
I'll try and reply the results.

Reply:
Unfortunately my several attempts failed to move that into modules hardware section. I suggest let's move on with this way for now while I'll try to work on that later. Ghaf on Orin Nx will be on use quite soon and there is still work to be finished to insert the firmware of ethernet.

Reply:
I'm just trying one last change which might work for moving all hardware options to modules. This might do the trick. If it also fails then at this step this might be the last trial for today.

Reply:
Finally solved moving device specific  part to modules/hardware/nvidia-jetson-orin/jetson-orin.nix on my last try. I'll commit that code now.

Reply:
Hi, I moved all hardware related options to modules/hardware/nvidia-jetson-orin/jetson-orin.nix. The changes isolated hardware options in modules part. Microvm works and this also solves the https://ssrc.atlassian.net/browse/SP-2585 issue. Formatting and all checks are done. I'll now add the ethernet firmware support for realtek 8168 so as to make netvm fully functional.